### PR TITLE
Primary analysis interface a11y fixes

### PR DESCRIPTION
**Pull request by dannon**
WIP, see individual commits for details.  Started with popping some fixes out of an out of date a11y testing configuration branch I've been working on (using playwright to run axe and spit reports for different areas in the app -- still forthcoming), and addressing some new stuff manually.  Targeting `WCAG 2.1 AAA`

Targeting primary analysis interface first in this pass, just the initial page load with a real(ish) history.

Before:
| Severity | Count|
| ------------- | ------------- |
| Critical | 42  |
| Serious  | 7  |
| Moderate | 61  |
| Minor | 0  |

After:
| Severity | Count|
| ------------- | ------------- |
| Critical | 0  |
| Serious  | 0  |
| Moderate | 1  |
| Minor | 0  |

The only issue left for full aaa compliance is related to incorrect styling of the iframe resulting from the recent routing/layout changes.  Working on options.

There three other `best practices` items that got flagged, but these are not requirements:

- Heading levels should only increase by one
    - We start with h4 in the tool/history panels, there is no h1.
- Ensure landmarks are unique
    - We have many `nav` items (content operations -- could be argued this shouldn't be a nav)
- Pages should contain a level-one heading
    - Maybe the masthead Galaxy should just be a styled h1?  I don't know that's particularly useful though.

## How to test the changes?
(Select all options that apply)
- [x] This is a refactoring of components with existing test coverage.

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: client/src/components/Panels/Buttons/PanelViewButton.vue [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
         v-b-tooltip.hover
         right
-        role="button"
         title="Show panel options"
         variant="link"
+        role="menu"
         aria-label="View all tool panel configurations"
         class="tool-panel-dropdown"
         size="sm">
+        <template v-slot:button-content>
```
Comment (mvdbeek) [inline client/src/components/Panels/Buttons/PanelViewButton.vue:11]: Nice, I didn't get how that was a problem with the checker website, that should fix it.

**Review by mvdbeek — APPROVED** [review]
Still WIP ?

**Comment by dannon** [comment]
@mvdbeek Yeah, there's one violation related to how we display the welcome I was hoping to get in, but it revealed a few problems with how we do routing that I'm working out.

**Comment by hexylena** [comment]
Just wanted to say how excited I am to see this! Thanks @dannon !

**Pull request merged**